clconf/template.go

```diff
@@ -125,17 +127,27 @@ func processTemplate(paths pathWithRelative, dest string, value interface{},
 		return result, fmt.Errorf("Error processing template: %v", err)
 	}
 
-	err = ioutil.WriteFile(target, []byte(content), mode)
-	if err != nil {
-		return result, err
-	}
-
-	if !options.KeepExistingPerms {
-		// WriteFile only uses the mode if the file is created during write
-		err = os.Chmod(target, mode)
+	if options.KeepEmpty || content != "" {
+		// This won't change the mode if it already exists
```

Comment:
this comment doesn't help.  the comment where you run another chmod basically tells the story.

Reply:
[Fixed](https://github.com/pastdev/clconf/pull/13/commits/e86bd2cdf541c247e3007ed37169507d2d56a096)